src/test/java/seedu/address/storage/JsonSerializableAgentAssistTest.java

```diff
@@ -26,7 +26,6 @@ public void toModelType_typicalPersonsFile_success() throws Exception {
                 JsonSerializableAgentAssist.class).get();
         AgentAssist agentAssistFromFile = dataFromFile.toModelType();
         AgentAssist typicalPersonsAgentAssist = TypicalPersons.getTypicalAgentAssist();
-        System.out.println("Agent Assist: " + agentAssistFromFile);
```

Comment:
May I ask why?

Reply:
I didn't realise you left that in as a debugging comment, let me add that back!